ctms/crud.py

```diff
@@ -79,25 +79,45 @@ def _contact_base_query(db):
     )
 
 
+def get_bulk_query(start_time, end_time, after_email_uuid, mofo_relevant):
+    if mofo_relevant is None:
+        return [
+            Email.update_timestamp >= start_time,
+            Email.update_timestamp < end_time,
+            Email.email_id != after_email_uuid,
+        ]
+    return [
+        Email.update_timestamp >= start_time,
+        Email.update_timestamp < end_time,
+        Email.email_id != after_email_uuid,
+        Email.mofo.has(mofo_relevant=mofo_relevant),
```

Comment:
`.has` was essential for truly getting the query to be properly created (printed sql query to visually confirm)